homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,26 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    int number = 1;
+    int number_prev = 0;
+    int temp = 0;
+    
+    if (sequence < 2) {
+        return sequence;
+    } 
+    else {
+        for (int i=2; i<=sequence; i++) {
+            temp = number + number_prev;
+            number_prev = number;
+            number = temp;
+        }
```

Comment:
Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..